src/platform/site-wide/user-nav/components/PersonalizationDropdown.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
1. now that this is a connected component, at the bottom of the file export the unconnected version as a named export for unit testing:

`export { PersonalizationDropdown }`

Reply:
2. then in the test that's choking on this (PersonalizationDropdown.unit.spec.jsx), change the import from `import PersonalizationDropdown from '../../components/PersonalizationDropdown';` to `import { PersonalizationDropdown } from '../../components/PersonalizationDropdown';` to use the unconnected version in the test